### PR TITLE
Bindu | AO-28 | Add entires for Bed Management and Operation Theater modules for RefApp 2.11

### PR DESCRIPTION
**Pull request by binduak**
Ticket Id
[https://issues.openmrs.org/browse/AO-28](https://issues.openmrs.org/browse/AO-28)

### CODE REVIEW

File: src/main/resources/add-ons-to-index.json [hunk 1]
```diff
@@ -6681,6 +6742,14 @@
...
           "version": "1.30.0"
         },
+        {
+          "uid": "org.openmrs.module.bedmanagement",
+          "version": "5.11.0"
+        },
+        {
+          "uid": "org.openmrs.module.operationtheater",
+          "version": "1.3.0"
+        },
```
Comment (ibacher) [inline src/main/resources/add-ons-to-index.json:6752]:
```suggestion
```

File: src/main/resources/add-ons-to-index.json [hunk 2]
```diff
@@ -1402,6 +1402,67 @@
...
       }
     },
+    {
+      "uid": "org.openmrs.module.bedmanagement",
+      "type": "OMOD",
+      "name": "Bed Management Module",
+      "maintainers": [
+        {
+          "name": "Ian Bacher"
+        },
```
Comment (ibacher) [inline src/main/resources/add-ons-to-index.json:1412]:
```suggestion
```

File: src/main/resources/add-ons-to-index.json [hunk 3]
```diff
@@ -1402,6 +1402,67 @@
...
+      }
+    },
+    {
+      "uid": "org.openmrs.module.operationtheater",
+      "type": "OMOD",
+      "name": "Operation Theater Module",
+      "maintainers": [
+        {
+          "name": "Ian Bacher"
+        },
```
Comment (ibacher) [inline src/main/resources/add-ons-to-index.json:1444]:
```suggestion
```

**Comment by ibacher** [comment]
Thanks @binduak! I've removed them from the Reference Application list only because that list is supposed to be the modules we ship with the RefApp and neither of these two are part of the OpenMRS Ref App.

**Comment by binduak** [comment]
sure. Thanks @ibacher

**Pull request merged**